cdk/lib/frontend.ts

```diff
@@ -117,71 +114,37 @@ export class Frontend extends GuStack {
       }),
     ];
 
-    const minimumProdInstances = 3;
-
     const ec2App = new GuEc2App(this, {
       applicationPort: 9000,
       app: "frontend",
       access: { scope: AccessScope.PUBLIC },
-      certificateProps: {
-        [Stage.CODE]: {
-          domainName:
-            "support.code.theguardian.com.origin.membership.guardianapis.com",
-          hostedZoneId: "Z1E4V12LQGXFEC",
-        },
-        [Stage.PROD]: {
-          domainName:
-            "support.theguardian.com.origin.membership.guardianapis.com",
-          hostedZoneId: "Z1E4V12LQGXFEC",
-        },
-      },
+      certificateProps,
       monitoringConfiguration: {
         noMonitoring: true,
       },
       userData,
       roleConfiguration: {
         additionalPolicies: policies,
       },
-      scaling: {
-        PROD: {
-          minimumInstances: minimumProdInstances,
-          maximumInstances: 6,
-        },
-        CODE: {
-          minimumInstances: 1,
-          maximumInstances: 2,
-        },
-      },
+      scaling,
       instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.SMALL),
     });
 
     // ---- Alarms ---- //
 
-    // TODO: remove [CDK] prefix. This prefix is needed as the alarms need to have
-    // a different name from the ones in the cloudformation template. We can remove
-    // this after the migration
```

Comment:
sneaking this in as I forgot to remove this comment when I removed the [CDK] prefix!